### PR TITLE
New package: bemenu-1.0.0

### PR DESCRIPTION
**Pull request by concatime**
How do I handle a doubled-license package?
https://github.com/Cloudef/bemenu/#license

### CODE REVIEW

**Comment by maxice8** [comment]
GPL-3.0-or-later, LGPL-3.0-or-later

File: common/shlibs [hunk 1]
```diff
@@ -2519,6 +2519,7 @@ libsignon-qt5.so.1 signond-8.58_1
 libsignon-plugins-common.so.1 signond-8.58_1
 libsignon-extension.so.1 signond-8.58_1
 libj4status-plugin.so.0 j4status-0.1_1
+libbemenu.so.1 bemenu-1.0.0_1
```
Comment (maxice8) [inline common/shlibs]: do you plan to add something that will use it in the near future ? if not please don't add common/shlibs entries

Comment (concatime) [reply common/shlibs]: Ok, I will remove it.

File: srcpkgs/bemenu/template [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+revision=1
+wrksrc=${pkgname}-master
+build_style=cmake
+configure_args="-DBUILD_TESTING=0"
+hostmakedepends="pkg-config $(vopt_if wayland wayland-devel)"
+makedepends="$(vopt_if curses ncurses-devel) cairo-devel pango-devel
+ $(vopt_if wayland libxkbcommon-devel) $(vopt_if x11 libXinerama-devel)"
+short_desc="Dynamic menu library and client program inspired by dmenu"
+maintainer="Issam Maghni <me@concati.me>"
+license="GPL-3.0-or-later,LGPL-3.0-or-later"
```
[the remark this answers is not in the document]
Comment (concatime) [reply srcpkgs/bemenu/template]: Done

**Comment by concatime** [comment]
Since there is no release, should I use `1.0.0` or `0.9.9`?

**Comment by maxice8** [comment]
You should ask upstream to make a tagged release we can use

**Comment by concatime** [comment]
Someone already asked https://github.com/Cloudef/bemenu/issues/16#issuecomment-83969356
So, `0.1.0`?

**Comment by maxice8** [comment]
> So, 0.1.0?

yes, when it is released

**Comment by maxice8** [comment]
No release yet ?

**Comment by lugubris** [comment]
v0.1.0 released
-> https://github.com/Cloudef/bemenu/issues/16#issuecomment-469052489

**Comment by maxice8** [comment]
Best solution is to make a pull request to update it, not to comment on a closed pull request

**Comment by maxice8** [comment]
Best solution is to make a pull request to update it, not to comment on a closed pull request

**Comment by concatime** [comment]
@lugubris If you look at the code, we’re already using `0.1.0`: https://github.com/void-linux/void-packages/blob/master/srcpkgs/bemenu/template

**Comment by lugubris** [comment]
Issam Maghni @ 2019-08-25 19:06 UTC:

> @lugubris If you look at the code, we’re already using 0.1.0: 
> https://
> github.com/void-linux/void-packages/blob/master/srcpkgs/bemenu/template

my bad, I didn't check the repository!

--
Lugubris
2AD2 612A AF91 A34B F09E
FB04 0EB1 022B D624 0D7A

**Pull request closed without merging**